client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -212,6 +200,12 @@ class PreviewFrame extends React.Component {
     this.addLoopProtect(sketchDoc);
     sketchDoc.head.insertBefore(consoleErrorsScript, sketchDoc.head.firstElement);
 
+    if (this.props.forceFullWidth) {
```

Comment:
I saw you reversed this, but maybe this should be removed entirely? Including the `forceFullWidth` prop?

Reply:
Not sure why it's still there, I thought I had already removed it!